components/src/core/Drawer/Drawer.tsx

```diff
@@ -8,18 +8,30 @@ import { InfoListItemProps as BaseInfoListItemProps } from '../InfoListItem';
 import { useDrawerLayout } from '../DrawerLayout/contexts/DrawerLayoutContextProvider';
 import { DrawerContext } from './DrawerContext';
 
-const useStyles = makeStyles({
-    paper: {
-        overflow: 'hidden',
-        position: 'inherit',
-    },
-    content: {
-        display: 'flex',
-        flexDirection: 'column',
-        height: '100%',
-        width: '100%',
-    },
-});
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            transition: theme.transitions.create('width', { duration: theme.transitions.duration.leavingScreen }),
+            minHeight: '100%',
+            '&$expanded': {
+                transition: theme.transitions.create('width', {
+                    duration: theme.transitions.duration.enteringScreen,
+                }),
+            },
+        },
+        expanded: {},
```

Comment:
New classes need to be added to the readme docs

Reply:
Do we want to add “expanded” as part of a user class too? (Not just “defaultClasses?)

Reply:
I would say yes, if it's a class that gets applied, we should give them a way to customize it if desired.